File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashSet <Integer> h = new HashSet <Integer>();
        for(int i=0;i<nums.length;i++){
            h.add(nums[i]);
        }
        return !(nums.length==h.size());
 
    }
}

File: Data Structures & Algorithms/find-duplicate-integer/submission-0.cpp
class Solution {
public:
    int findDuplicate(vector<int>& nums) {
        int idx = 0;
        for(int x:nums){
            idx = abs(x)-1;
            if(nums[idx]<0){
                return abs(x);
            }else{
                nums[idx]*=-1;
            }
        }
        return 0;
        
    }
};
